recipes/ruby/build.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+set -x
+
+./configure --prefix=$PREFIX --disable-install-doc --enable-load-relative \
+  --enable-shared --with-openssl-dir=$PREFIX --with-readline-dir=$PREFIX \
+  --with-tcl-dir=$PREFIX --with-tk-dir=$PREFIX --with-libyaml-dir=$PREFIX \
+  --with-zlib-dir=$PREFIX
```

Comment:
Is there a `--with-gmp-dir` argument like the other ones? Might be worth adding as a preventive measure. Though it does seem to be linking correctly without it.

Also a small thing, would recommend quoting environment variables. Just in case we end up with spaces in the path in the future.